Objects/listobject.c

```diff
@@ -215,6 +215,12 @@ list_new_prealloc(Py_ssize_t size)
     return (PyObject *) op;
 }
 
+PyObject *
+_PyList_NewPrealloc(Py_ssize_t size)
```

Comment:
Instead of adding a new function `_PyList_NewPrealloc` with identical functionality as `list_new_prealloc`, why not just rename `list_new_prealloc` to `_PyList_NewPrealloc`?